jsk_interactive_markers/jsk_interactive_marker/src/marker_6dof.cpp

```diff
@@ -177,15 +186,29 @@ class Marker6DOF {
       object_marker.color.b = object_b_;
       object_marker.color.a = object_a_;
       object_marker.pose.orientation.w = 1.0;
-    }else if(object_type_ == std::string("line")){
+    }
```

Comment:
because there is indent here https://github.com/jsk-ros-pkg/jsk_visualization/pull/657/files/13b5a38ddf4c47c30930e387da2c1632b12b49ab#diff-51e3d57b1c1b9f815cd7be461b98acb4R178 .